app/components/typography/itemsList/ItemList.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from "react";
+
+interface ItemListProps {
+  items: string[];
+  color: string;
+  fontWeight: string;
+  fontSize: string;
+  lineHeight: string;
+}
+
+const ItemList: FC<ItemListProps> = ({
+  items,
+  color,
+  fontWeight,
+  fontSize,
+  lineHeight,
```

Comment:
Instead of color, fontWeight, fontSize, and lineHeight, consider consolidating these into a single className prop. This change simplifies the props and makes it easier for users to apply any Tailwind CSS classes.
```suggestion
  className = ""
```